source/upgrade.txt

```diff
@@ -76,6 +76,16 @@ Version 6.x Breaking Changes
 - Automatic Encryption methods are now in the {+driver-short+}. You must
   import these methods from the driver instead of from
   ``mongodb-client-encryption``.
+- Removed the ``ObjectId`` constructor that accepted a 12-character string.
+- The ``abortTransaction()`` and ``commitTransaction()`` methods return
+  ``null`` instead of the raw command results.
+- Removed connection option helpers that accepted values other than ``true``
+  or  ``false`` as booleans. You must provide either ``true`` or ``false`` values in
+  the connection string or to the MongoClient constructor.
+- Removed the Binary BSON type constructor that accepted a string.
+- The ``Binary.write()`` method no longer accepts a string to write to the binary
```

Comment:
Question:
Is this similar to the "Binary BSON type"? If so, I think it could be good to keep consistent formatting. If it is not related, I think further detail on how it is different could be helpful.